addOns/ascanrules/src/main/java/org/zaproxy/zap/extension/ascanrules/RemoteCodeExecutionCve20121823ScanRule.java

```diff
@@ -68,11 +69,17 @@ public class RemoteCodeExecutionCve20121823ScanRule extends AbstractAppPlugin {
             PAYLOAD_BOILERPLATE.replace("<<<<COMMAND>>>>", "cmd.exe /C echo " + RANDOM_STRING);
     private static final String NIX_PAYLOAD =
             PAYLOAD_BOILERPLATE.replace("<<<<COMMAND>>>>", "echo " + RANDOM_STRING);
-    private static final Map<String, String> ALERT_TAGS =
-            CommonAlertTag.toMap(
-                    CommonAlertTag.OWASP_2021_A06_VULN_COMP,
-                    CommonAlertTag.OWASP_2017_A09_VULN_COMP,
-                    CommonAlertTag.WSTG_V42_INPV_12_COMMAND_INJ);
+    private static final String CVE = "CVE-2012-1823";
+    private static final Map<String, String> ALERT_TAGS = new HashMap<>();
+
+    static {
+        ALERT_TAGS.putAll(
+                CommonAlertTag.toMap(
+                        CommonAlertTag.OWASP_2021_A06_VULN_COMP,
+                        CommonAlertTag.OWASP_2017_A09_VULN_COMP,
+                        CommonAlertTag.WSTG_V42_INPV_12_COMMAND_INJ));
+        ALERT_TAGS.put(CVE, "");
```

Comment:
Shouldnt the CVE tags include a link (inc the one above), eg to https://nvd.nist.gov/vuln/detail/cve-2012-1823
I know https://www.zaproxy.org/alerttags/cve-2022-42889/ doesnt, but I think that one should as well ...

Reply:
I guess we could

Reply:
Shall we merge this and I’ll do another PR to linkify things?

Reply:
I'm good with that 😁

Reply:
Thanks